Jenkinsfile

```
@@ -24,6 +25,7 @@ pipeline {
   }
   parameters {
     string name: 'EE_DOWNSTREAM', defaultValue: 'cambpm-ee-main-pr/' + cambpmDefaultBranch(), description: 'The name of the EE branch/PR to run the EE pipeline on, e.g. cambpm-ee-main/PR-333'
+    booleanParam name: 'HAS_ARTIFACTS', defaultValue: false, description: 'whether stores the artifacts in the build'
```

The name and description were a bit confusing for me first. I think something like this would make it easier to understand what the flag does. What do you think?

```suggestion
    booleanParam name: 'HAS_ARTIFACTS', defaultValue: false, description: 'Flag indicating if build has artifacts that need to be archived.'
```

---

The param is used only to store temporary the artifacts for testing. It will be removed before merge.